logstash-core/lib/logstash/runner.rb

```diff
@@ -82,6 +82,10 @@ class LogStash::Runner < Clamp::Command
     I18n.t("logstash.runner.flag.auto_reload"),
     :attribute_name => :auto_reload, :default => false
 
+  option ["-ap", "--http-port"], "WEB_API_HTTP_PORT",
```

Comment:
+1, no 2 char flags. we don't even need a single char flag. Just `--http-port` will do